src/XrdPfc/XrdPfcIO.hh

```diff
@@ -77,6 +77,7 @@ private:
 
    void SetInput(XrdOucCacheIO*);
 
+protected:
```

Comment:
Could make only the new things protected? This is to avoid exposing previously private parts of the class unnecessarily.